src/java/org/apache/cassandra/guardrails/GuardrailsConfig.java

```diff
@@ -165,6 +168,11 @@ public void applyConfig()
         enforceDefault(tombstone_warn_threshold, v -> tombstone_warn_threshold = v, 1000, 1000);
         enforceDefault(tombstone_failure_threshold, v -> tombstone_failure_threshold = v, 100000, 100000);
 
+        // Default to no warning and failure at 4 times the maxTopK value
+        int maxTopK = CassandraRelevantProperties.SAI_VECTOR_SEARCH_MAX_TOP_K.getInt();
```

Comment:
Unrelated to the changes, but at some point we should make `SAI_VECTOR_SEARCH_MAX_TOP_K` a guardrail.

Reply:
Agreed